metaflow/plugins/kfp/kfp.py

```diff
@@ -705,6 +705,21 @@ def pipeline_transform(op: ContainerOp):
                             ),
                         )
                     )
+                # add in env variable for ServiceAccount for spark
```

Comment:
Can you please add in the comment `add in env variable for ServiceAccount for Zillow Spark solution`?

Reply:
yep